### PR TITLE
chore: added cloudbuild test and deploy scripts

### PR DESCRIPTION
**Pull request by danted22**
Needs the terraform setup to be created first

### CODE REVIEW

**Comment by danted22** [comment]
Part of https://github.com/sendahug/internal/issues/21

**Comment by shirblc** [comment]
Just so we don't forget: we also need to add a docker file in this PR...

**Review by danted22 — COMMENTED** [review]
This is good to go!

**Review by shirblc — APPROVED** [review]

**Review by shirblc — CHANGES_REQUESTED** [review]
I think there's a missing Changelog? 🤨🧐

**Comment by danted22** [comment]
Oops there is. Will add after twins bedtime!

**Comment by danted22** [comment]
Added 😀

File: changelog/pr626.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "pr_number": 626,
+    "changes": [
+        {
+            "change": "Chores",
+            "description": "Added deployment to cloud run scripts for staging and live environments"
+        },
+        {
+            "change": "Chores",
+            "description": "Added Dockerfile for containserszation of the app for deployment"
```
Comment (shirblc) [inline changelog/pr626.json]: I think that was supposed to be containerisation 😉

**Review by shirblc — APPROVED** [review]

**Pull request merged**